translations/CONTRIBUTING.mal.md

```diff
@@ -0,0 +1,69 @@
+# ഓപ്പൺഇബിഎസിലേക്ക് സംഭാവന ചെയ്യുന്നു
+
+കൂടുതൽ ഓപ്പൺഇബിഎസ് ഹാക്കർമാരെ ഞങ്ങൾ എപ്പോഴും അന്വേഷിച്ചുകൊണ്ട് ഇരിക്കുന്നു . ഇവിടെ പറഞ്ഞിരിയ്ക്കുന്ന [അവലോകനം](./contribute/design/README.md) വായിച്ചുകൊണ്ട് നിങ്ങൾക്ക് സംഭാവനകൾ  ആരംഭിക്കാൻ കഴിയും.
+
+ഒന്നാമതായി, നിങ്ങൾക്ക് എന്തെങ്കിലും ഉറപ്പോ ഭയമോ ഉണ്ടെങ്കിൽ, പ്രശ്നം ചോദിക്കുക അല്ലെങ്കിൽ സമർപ്പിക്കുക അല്ലെങ്കിൽ എന്തായാലും അഭ്യർത്ഥന വലിക്കുക. നിങ്ങളുടെ മികച്ച ശ്രമം നടത്തിയതിന് നിങ്ങൾ ആക്രോശിക്കില്ല. സംഭവിക്കാവുന്ന ഏറ്റവും മോശം കാര്യം എന്തെങ്കിലും മാറ്റാൻ നിങ്ങളോട് മാന്യമായി ആവശ്യപ്പെടും എന്നതാണ്. ഏതെങ്കിലും തരത്തിലുള്ള സംഭാവനകളെ ഞങ്ങൾ അഭിനന്ദിക്കുന്നു, ഒപ്പം നിയമങ്ങളുടെ ഒരു മതിൽ ലഭിക്കാൻ ഞങ്ങൾ ആഗ്രഹിക്കുന്നില്ല.
+
+എന്നിരുന്നാലും, പ്രോജക്റ്റിലേക്ക് സംഭാവന ചെയ്യുന്നതിനുള്ള മികച്ച മാർഗ്ഗത്തെക്കുറിച്ച് കുറച്ചുകൂടി മാർഗ്ഗനിർദ്ദേശം ആഗ്രഹിക്കുന്ന വ്യക്തികൾക്കായി, വായിക്കുക. നിങ്ങളുടെ സംഭാവനകളിൽ ഞങ്ങൾ അന്വേഷിക്കുന്ന എല്ലാ പോയിന്റുകളും ഈ പ്രമാണം ഉൾക്കൊള്ളുന്നു, നിങ്ങളുടെ സംഭാവനകളെ വേഗത്തിൽ ലയിപ്പിക്കുന്നതിനോ അഭിസംബോധന ചെയ്യുന്നതിനോ ഉള്ള സാധ്യതകൾ ഉയർത്തുന്നു.
+
+ഓപ്പൺ സോഴ്‌സിലെ ഒരു പുതുമയാണ് ഓപ്പൺഇബിഎസ് എന്ന് അത് പറഞ്ഞു. നിങ്ങൾക്ക് കഴിയുന്ന വിധത്തിൽ സംഭാവന ചെയ്യാൻ നിങ്ങൾക്ക് സ്വാഗതം, ഒപ്പം നൽകിയ എല്ലാ സഹായങ്ങളും വളരെ വിലമതിക്കപ്പെടുന്നു. 
+
+- [പുതിയ പ്രവർത്തനം അഭ്യർത്ഥിക്കുന്നതിനോ ഡോക്യുമെന്റേഷൻ പരിഹരിക്കുന്നതിനോ ബഗുകൾ റിപ്പോർട്ടുചെയ്യുന്നതിനോ പ്രശ്നങ്ങൾ ഉയർത്തുക.](#ഉയർത്തുന്ന-പ്രശ്നങ്ങൾ)
+- [ഡോക്യുമെന്റേഷൻ മെച്ചപ്പെടുത്തുന്നതിന് മാറ്റങ്ങൾ സമർപ്പിക്കുക.](#സമർപ്പിക്കുക-മാറ്റുക-മെച്ചപ്പെടുത്തുക-ഡോക്യുമെന്റേഷൻ) 
+- [പുതിയ സവിശേഷതകൾ / മെച്ചപ്പെടുത്തലുകൾക്കായി നിർദ്ദേശങ്ങൾ സമർപ്പിക്കുക.](#പുതിയ സവിശേഷതകൾക്കായി സമർപ്പിക്കുക-നിർദ്ദേശങ്ങൾ)
+- [ഡോക്യുമെന്റേഷൻ അല്ലെങ്കിൽ കോഡുമായി ബന്ധപ്പെട്ട നിലവിലുള്ള പ്രശ്നങ്ങൾ പരിഹരിക്കുക.](#ഉറവിടത്തിലേക്ക് കോഡ്-ബഗ് പരിഹാരങ്ങൾ സംഭാവന ചെയ്യുന്നു)
+
+നിങ്ങളുടെ ഹാക്കുകൾ നൽകുന്നതിനുമുമ്പ് നിങ്ങൾ പാലിക്കേണ്ട ചില ലളിതമായ മാർഗ്ഗനിർദ്ദേശങ്ങൾ ഉണ്ട്.
+
+## പ്രശ്നങ്ങൾ ഉയർത്തുന്നു
+
+പ്രശ്നങ്ങൾ ഉന്നയിക്കുമ്പോൾ, ഇനിപ്പറയുന്നവ വ്യക്തമാക്കുക:
+- നിരൂപകന് പരിശോധിക്കുന്നതിന് ഇഷ്യു ടെംപ്ലേറ്റിൽ വ്യക്തമാക്കിയ സജ്ജീകരണ വിശദാംശങ്ങൾ പൂരിപ്പിക്കേണ്ടതുണ്ട്.
+- പ്രശ്നം സംഭവിച്ച ഒരു രംഗം (അത് എങ്ങനെ പുനർനിർമ്മിക്കാം എന്നതിനെക്കുറിച്ചുള്ള വിശദാംശങ്ങൾക്കൊപ്പം).
+- സോഫ്റ്റ്വെയർ പ്രദർശിപ്പിക്കുന്ന പിശകുകളും ലോഗ് സന്ദേശങ്ങളും.
+- ഉപയോഗപ്രദമായേക്കാവുന്ന മറ്റേതെങ്കിലും വിശദാംശങ്ങൾ.
+
+
+## ഡോക്യുമെന്റേഷൻ മെച്ചപ്പെടുത്തുന്നതിന് മാറ്റം സമർപ്പിക്കുക
+
+ഡോക്യുമെന്റേഷൻ ശരിയായി ലഭിക്കുന്നത് ബുദ്ധിമുട്ടാണ്! ഇത് റഫർ ചെയ്യുക [പേജ്](./contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md) ഉചിതമായ ടാഗുകൾ ഉപയോഗിച്ച് പുൾ അഭ്യർത്ഥനകൾ സമർപ്പിച്ചുകൊണ്ട് ഡവലപ്പർ ഡോക്യുമെന്റേഷൻ എങ്ങനെ മെച്ചപ്പെടുത്താം എന്നതിനെക്കുറിച്ചുള്ള കൂടുതൽ വിവരങ്ങൾക്ക്. ഇവിടെ ഒരു [ടാഗുകളുടെ പട്ടിക](./contribute/labels-of-issues.md) അത് അതിനായി ഉപയോഗിക്കാം. ഞങ്ങളുടെ ഡോക്യുമെന്റേഷൻ വൃത്തിയുള്ളതും മനസിലാക്കാൻ എളുപ്പമുള്ളതും ആക്സസ് ചെയ്യാവുന്നതുമായി സൂക്ഷിക്കാൻ ഞങ്ങളെ സഹായിക്കുക.
```

Comment:
Fix the hyperlink with the correct word.

Reply:
Also, can you reorganise the sentence.

Reply:
corrected as per the suggessions